### PR TITLE
Synced repo + added again a bunch of fixes

### PR DESCRIPTION
**Pull request by d4f5409d**
### 🇺🇸🇬🇧 This pull request is based on our Hungarian community's suggestions. 

_If you want to participate discussing the current state of Hungarian translation and where we will head towards in the future, please join our Matrix.org room here: **#organicmapstranslate_hu:matrix.org** or https://matrix.to/#/!kzBidywONsbbZNRSrP:matrix.org_

#### This pull request contains:
- Sound navigation improvements
- Filling up missing translations
- Correcting translations to represent what they really mean in the original version
- Fixing typos, mistyped words

---

### 🇭🇺 Ez a pull request a magyar közösségünk javaslatain alapul. 

_Ha részt szeretnél  venni a magyar fordítás jelenlegi helyzetének megvitatásában és abban, hogy merre haladjunk a jövőben, kérjük, csatlakozz a Matrix.org szobánkhoz itt: **#organicmapstranslate_hu:matrix.org** vagy itt https://matrix.to/#/!kzBidywONsbbZNRSrP:matrix.org_

#### Ez a pull request a következőket tartalmazza:

- Hangnavigációs javítások
- Hiányzó fordítások lefordítása
- Fordítások javítása, hogy azt képviseljék, amit az eredeti szövegben jelentenek
- Gépelési hibák, elírt szavak javítása

Signed-off-by: @d4f5409d <76624594+d4f5409d@users.noreply.github.com>
> You are free and pleased to use my template for every translation related pull request.

### CODE REVIEW

**Review by vng — APPROVED** [review]

**Review by aronkvh — APPROVED** [review]
Great work! köszi

File: data/strings/strings.txt [hunk 1]
```diff
@@ -2191,6 +2191,7 @@
     ru = Внутренний скрытый накопитель
     tr = Dahili özel depolama
     uk = Внутрішнє приватне сховище
+    hu = Belső privát tárolóhely
```
Comment (biodranik) [inline data/strings/strings.txt:2194]: It would be great to add it in alphabet order (en at the top is the exception) here and in other places.

**Review by biodranik — COMMENTED** [review]
Good job, thanks! Please also sign the commit (the easiest way is to use `git commit --amend -s` from the command line).

File: data/strings/strings.txt (hunk 1 above)
Comment (d4f5409d) [reply data/strings/strings.txt:2194]: Next time.

Comment (biodranik) [reply data/strings/strings.txt:2194]: If you didn't learn yet how to amend commits and force-push, then you can create a second signed commit where necessary strings will be moved to the appropriate place. We'll squash them for you.

Learning git rebase and amend flow is highly recommended ;-)

**Comment by rtsisyk** [comment]
@d4f5409d 
```
git commit --amend -s
git push origin -f
```

**Comment by d4f5409d** [comment]
> @d4f5409d
> 
> ```
> git commit --amend -s
> git push origin -f
> ```

GitHub's codespaces feature really made for me this easier. cool.

**Comment by biodranik** [comment]
Please check the latest version of strings.txt, you're modifying a very old outdated version.

**Comment by d4f5409d** [comment]
> Please check the latest version of strings.txt, you're modifying a very old outdated version.

No, I synced literally a week ago, what are you talking about?

**Comment by d4f5409d** [comment]
April fools is over, isn't it? 😅

**Comment by biodranik** [comment]
`git log 89dd640da35082abbe5d32623851ef0bc50e6faf` for your first commit shows that its parent commit was done on May 21, 2022. You are modifying a very outdated version of strings.txt.

```
commit 89dd640da35082abbe5d32623851ef0bc50e6faf
Author: d4f5409d <76624594+d4f5409d@users.noreply.github.com>
Date:   Fri Mar 10 20:40:50 2023 +0100

    Quick fix Hungarian translations and add more to it
    
    Signed-off-by: d4f5409d <76624594+d4f5409d@users.noreply.github.com>

commit 4ce6aafdc6d7cf8c67df4ac8fe57ded00f9aa722
Author: dbf <dbf256@gmail.com>
Date:   Sat May 21 09:04:57 2022 +0300

    [docs] Fix typo
    Signed-off-by: dbf <dbf256@gmail.com>
```

**Comment by d4f5409d** [comment]
> `git log 89dd640da35082abbe5d32623851ef0bc50e6faf` for your first commit shows that its parent commit was done on May 21, 2022. You are modifying a very outdated version of strings.txt.
> 
> ```
> commit 89dd640da35082abbe5d32623851ef0bc50e6faf
> Author: d4f5409d <76624594+d4f5409d@users.noreply.github.com>
> Date:   Fri Mar 10 20:40:50 2023 +0100
> 
>     Quick fix Hungarian translations and add more to it
>     
>     Signed-off-by: d4f5409d <76624594+d4f5409d@users.noreply.github.com>
> 
> commit 4ce6aafdc6d7cf8c67df4ac8fe57ded00f9aa722
> Author: dbf <dbf256@gmail.com>
> Date:   Sat May 21 09:04:57 2022 +0300
> 
>     [docs] Fix typo
>     Signed-off-by: dbf <dbf256@gmail.com>
> ```

Yes, but this pull request has been made because I synced my forked repo. I think just ignore it, either way I checked, the strings are still not present in OM today.

**Comment by biodranik** [comment]
It is important to get fresh strings. But we can't merge them if you don't sync strings.txt file and fix merge conflicts.

I would recommend you:
1. To get the fresh master branch in your github fork.
2. Check these strings in the fresh version of strings.txt and edit those that are still missing/need to be updated.
3. Force-push an updated branch here in this PR.

If you find it complicated, you may download the latest strings.txt file from our master branch, edit it manually, and share it here with us, so we can create an appropriate Pull request. But for a longer-term benefit, it would be great to learn git and how to handle basic tasks:
- sync
- amending commits
- interactive git rebase
- force-push branches to the same PRs

**Pull request closed without merging**